source/databricks/calculation-engine/tests/integration/calculator/calculator_job/conftest.py

```diff
@@ -96,12 +114,68 @@ def test_data_written_to_delta_tables(
     )
 
 
+@pytest.fixture(scope="session")
+def charge_input_data_written_to_delta_tables(
+    spark: SparkSession,
+    test_files_folder_path: str,
+    input_database_created: None,
+    data_lake_path: str
+) -> None:
+    # Charge master data periods
+    spark.sql(
+        f"CREATE TABLE IF NOT EXISTS {infra.INPUT_DATABASE_NAME}.{infra.CHARGE_MASTER_DATA_PERIODS_TABLE_NAME} USING DELTA \
+            LOCATION '{data_lake_path}/calculation_input/charge_master_data_periods'"
+    )
+
+    charge_master_data_periods_df = spark.read.csv(
+        f"{test_files_folder_path}/ChargeMasterDataPeriods.csv",
+        header=True,
+        schema=charge_master_data_periods_schema,
+    )
+
+    charge_master_data_periods_df.write.format("delta").mode("overwrite").option("overwriteSchema", "true").saveAsTable(
+        f"{infra.INPUT_DATABASE_NAME}.{infra.CHARGE_MASTER_DATA_PERIODS_TABLE_NAME}"
+    )
```

Comment:
Applies to all similar: How about using a reusable function like this?
```suggestion
    write_test_data_to_table(
        f"{test_files_folder_path}/ChargeMasterDataPeriods.csv",
        f"{infra.INPUT_DATABASE_NAME}.{infra.CHARGE_MASTER_DATA_PERIODS_TABLE_NAME}"
        charge_master_data_periods_schema,
    )
```